src/components/Form/ContactForm.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import Style from "./ContactForm.module.css";
+import { GrMail } from "react-icons/gr";
+import Button from "../../components/Button";
+
+const ContactForm = () => {
+    const [name, setName] = useState("");
```

Comment:
You can use a single state with these parameters as object and call a single setFunction when changing the inputfield values